.gitignore

```diff
@@ -1,5 +1,6 @@
 **/.DS_Store
 .vscode
+.palate
```

Comment:
I keep a small text file in my Atom editor called `.palate` where I keep code snippets like the basic template for codes in the GMF:

```
{
  "system": "",
  "code": "",
  "display": ""
}
```